WalletWasabi.Gui/Tabs/AboutViewModel.cs

```diff
@@ -73,13 +74,34 @@ public override void OnOpen(CompositeDisposable disposables)
 		{
 			base.OnOpen(disposables);
 
-			var updateChecker = UpdateChecker;
-			if (updateChecker is { })
+			var global = Locator.Current.GetService<Global>();
+			var hostedServices = global.HostedServices;
+
+			this.WhenAnyValue(x => x.UpdateChecker)
+				.Where(x => x is { })
+				.Take(1)
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(x =>
+				{
+					Observable.FromEventPattern<UpdateStatus>(x, nameof(x.UpdateStatusChanged))
+						.ObserveOn(RxApp.MainThreadScheduler)
+						.Subscribe(e => CurrentBackendMajorVersion = WasabiClient.ApiVersion.ToString())
+						.DisposeWith(disposables);
+
+					CurrentBackendMajorVersion = WasabiClient.ApiVersion.ToString();
```

Comment:
I'm confused. What would be different if the method would simply be:

```csharp
public override void OnOpen(CompositeDisposable disposables)
{
	base.OnOpen(disposables);
	CurrentBackendMajorVersion = WasabiClient.ApiVersion.ToString();
}
```

?

Given that currently I can see that `WasabiClient.ApiVersion` is a static property computed from a constant:

```csharp
public static ushort ApiVersion { get; private set; } = ushort.Parse(Constants.BackendMajorVersion);
```

Or how come that `UpdateChecker` comes into play?

Reply:
No, it is not silly. That complex plumbing is for updating the Api version in case it change. Remove all this stuff completely and let is as simple as the @kiminuo snippet in his question above.

Reply:
Yes, it was about to update the version on change. Now you have to close/open About to see if there was an update. API version not changing only at the initialization and opening About at the beginning makes not much sense. So I removed the stuff.

Reply:
> I'm confused. What would be different if the method would simply be:
> 
> ```cs
> public override void OnOpen(CompositeDisposable disposables)
> {
> 	base.OnOpen(disposables);
> 	CurrentBackendMajorVersion = WasabiClient.ApiVersion.ToString();
> }
> ```
> 
> ?
> 
> Given that currently I can see that `WasabiClient.ApiVersion` is a static property computed from a constant:
> 
> ```cs
> public static ushort ApiVersion { get; private set; } = ushort.Parse(Constants.BackendMajorVersion);
> ```
> 
> Or how come that `UpdateChecker` comes into play?

It can change later if 
https://github.com/zksnacks/WalletWasabi/blob/b08a362380ad813d110a2776cc3f3c8fb0ed73e1/WalletWasabi/WebClients/Wasabi/WasabiClient.cs#L266